dialogue-clients/metrics.md

```diff
@@ -7,6 +7,7 @@
 ### dialogue.client
 Dialogue client response metrics provided by the Apache client channel.
 - `dialogue.client.response.leak` tagged `client-name`, `service-name`, `endpoint` (meter): Rate that responses are garbage collected without being closed. This should only occur in the case of a programming error.
+- `dialogue.client.response.delta` tagged `client-name` (timer): Difference in request time reported by the client and by the server. This metric is only reported when the remote server provides a `Server-Timing` response header with a timing value for `server`.
```

Comment:
naming is hard, but maybe "networkoverhead" "servertimingoverhead" or "servertimingdelta" ?

Might also be worth reminding people that the timing value for `server` needs to be in milliseconds?

Reply:
servertimingoverhead sounds much better, thanks!

> Might also be worth reminding people that the timing value for server needs to be in milliseconds?

I don't think we should mention units here since all our timers report values normalized to microseconds, I don't want to confuse folks.

Reply:
I meant in the header string, in `Server-Timing: server;dur=50` the 50 represents milliseconds